docs/dev-guide/mobile-feature-flags.md

```diff
@@ -20,6 +20,7 @@ Here are the currently implemented flags:
 * `invite.enabled`: Flag indicating if invite functionality should be enabled. Default: enabled (true).
 * `ios.recording.enabled`: Flag indicating if recording should be enabled in iOS. Default: disabled (false).
 * `ios.screensharing.enabled`: Flag indicating if screen sharing should be enabled in iOS. Default: disabled (false).
+* `android.screensharing.enabled`: Flag indicating if screen sharing should be enabled in android. Default: disabled (true).
```

Comment:
Please sort it alphabetically. Also the text is wrong, it should say "Default: enabled (true)."